docs/proposals/object-tracking.md

```diff
@@ -0,0 +1,158 @@
+- [Object Tracking Service](#object-tracking-service)
+  - [Motivation](#motivation)
+    - [Goals](#goals)
+  - [Proposal](#proposal)
+    - [Use Cases](#use-cases)
+  - [Design Details](#design-details)
+    - [CRD API Group and Version](#crd-api-group-and-version)
+    - [Object tracking service CRD](#object-tracking-service-crd)
+    - [Object tracking service type definition](#object-tracking-service-type-definition)
+      - [Validation](#validation)
+    - [Object tracking service sample](#object-tracking-service-sample)
+  - [Controller Design](#controller-design)
+    - [Object tracking service Controller](#object-tracking-service-controller)
+    - [Downstream Controller](#downstream-controller)
+    - [Upstream Controller](#upstream-controller)
+    - [Details of api between GM(cloud) and LC(edge)](#details-of-api-between-gmcloud-and-lcedge)
+    - [Flow of object tracking service creation](#flow-of-object-tracking-service-creation)
+  - [Workers Communication](#workers-communication)
+# Object Tracking Service
+## Motivation
+Object tracking is an important technology in the field of computer vision, which is widely used in security monitoring, intelligent transportation and other fields. Generally, object tracking applications have high latency requirements, which cannot be met by traditional cloud computing schemes. The target tracking schemes based on edge computing architecture have the characteristics of low latency and data privacy security, and are the mainstream technology trend in the industry. However, how to efficiently use the distributed resources of edge computing to improve throughput and accuracy, and reduce time delay is the current technical difficulty. We propose the first open source end-to-end multi-edge collaborative target tracking application implementation solution. Based on KubeEdge's cloud-edge collaboration and resource management capabilities, we use multiple edge computing nodes to execute the AI inference tasks of target tracking workflow in parallel to solve the above problems.
```

Comment:
we can indicate that multi-edge in vision scenario will bring accuracy promotion,  no only latency. its a same problem for PR of "object search"

Reply:
I revised the Motivation and added relavent description: "Multi-edge collaborative computing can accelerate large-scale object search applications and improve the search accuracy, which is the future development trend....".  The other proposals is added too.